ansible-scylla-node/tasks/handle_ear_keys.yml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
This playbook (not a task!) handles "local table keys" AFAIU.
You need to be very explicit about it in the patch description.

In addition to that there are also "System keys" (https://enterprise.docs.scylladb.com/stable/operating-scylla/security/encryption-at-rest.html#system-keys), which may be configured **in addition** to "local table keys" according to my understanding of https://enterprise.docs.scylladb.com/stable/operating-scylla/security/encryption-at-rest.html#table-keys